cmd/ltctl/main.go

```diff
@@ -49,12 +49,39 @@ func RunCreateCmdF(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func destroyAllButMetrics(config deployment.Config) error {
+	// Override all created resources so we destroy everything but the metrics
+	// instance.
+	config.DestroyAllButMetrics()
```

Comment:
Now that I see it in action, I find the naming a bit confusing. Could we revisit? Something like `MarkForDestroyAllButMetrics` perhaps.

Reply:
That's a good name, thanks! I couldn't come up with one that I liked, the `Mark` verb is key :ok_hand: Done in https://github.com/mattermost/mattermost-load-test-ng/pull/905/commits/1882968fec784af7dc4346de652c29f3c0a15969.